Src/CSharpier.Tests/TestFiles/AttributeList/AttributeLists.cst

```diff
@@ -50,4 +50,17 @@ class ClassName
         "11111111111111111111111111",
         "22222222222222222222222222")]
     void LongAttribute() { }
+
+    public void MethodName([FromBody] SomeParameter parameter)
+    {
+        return;
+    }
+
+    public void MethodName(
+        int someParameter,
+        [LongAttributeName(SomeFlag.SomeValue)]
+        AnotherObject anotherObject
```

Comment:
Minor point: you reckon we should indent this?
To show at a glance that this line continues the previous line's parameter instead of starting a new one?